### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by poofywings** [derfl007/BnBSellItems#1]
Hello.

Currently, this mod is not working for me. I've updated the BepInEx to the most recent version and even redownloaded and reinstalled the BnBSellItems.dll.
Whenever I trash an item, ingredient, etc, I don't get any gold or trash back. I'm not sure if any of the recent Bear n Breakfast updates broke the mod. I hope you can assist at your earliest convenience. Using your mod greatly improves the quality of life in the game.

Thanks.

**Comment by M4DMAXX** [comment]
Not a Bug, just outdated Mod, derfl would need to update it.

**Comment by derfl007** [comment]
Yeah the demand for mods was extremly low, my two mods are still the only ones on nexus.

Might look into it, but not sure if it's worth it :(

**Comment by M4DMAXX** [comment]
I understand that, just got Bear and Breakfast trough Epic, and Nexus was my first Site to look for Mods :D

Too bad that there is not much demand for Mods. And I'm a Gamer since 17 years, but never got into programming or modding on my own, just used Mods and only had ported a few Skyrim Mods from LE to SE, and it takes me a lot of time and headache xD

I have high respect for any Modder out there.
